micrometer-core/src/main/java/io/micrometer/core/instrument/cumulative/CumulativeDistributionSummary.java

```diff
@@ -15,16 +15,23 @@
  */
 package io.micrometer.core.instrument.cumulative;
 
-import com.google.common.util.concurrent.AtomicDouble;
 import io.micrometer.core.instrument.AbstractDistributionSummary;
 import io.micrometer.core.instrument.Clock;
 import io.micrometer.core.instrument.Measurement;
 import io.micrometer.core.instrument.Statistic;
 import io.micrometer.core.instrument.histogram.HistogramConfig;
+import io.micrometer.core.instrument.util.AtomicDouble;
```

Comment:
Good catch! I think you should use `java.util.concurrent.atomic.DoubleAdder` instead

Reply:
@checketts Thanks for the feedback! `DoubleAdder` came to mind first but I didn't figure out how to replace `compareAndSet()` with it. Any suggestion?

Reply:
@checketts I updated this PR in e785aaf as you suggested but for `max` I tried another way.